pom.xml

```diff
@@ -54,11 +54,11 @@
 
     <opensaml.version>5.0.0</opensaml.version>
     <shib.support.version>9.0.0</shib.support.version>
```

Comment:
Propose to upgrade opensaml to 5.1.2 to avoid vulnerabilities
This also raises shib.support.version to 5.1.2 and jackson version to 2.16.1

Reply:
Let's do that in another PR. I need to analyze what is new in OpenSAML 5.1.X.